src/tests/custom-component/custom-component.stories.tsx

```diff
@@ -0,0 +1,96 @@
+import {DEFAULT_RENDER_CONFIGURATION, RenderForm} from '@lib/renderer';
+import {DEFAULT_VALIDATORS} from '@lib/validation';
+import {expect} from '@storybook/jest';
+import type {ComponentStory, Meta} from '@storybook/react';
+import {userEvent, within} from '@storybook/testing-library';
+import {IComponentProps} from '@types';
+import React from 'react';
+
+const meta: Meta<typeof RenderForm> = {
+  title: 'Usage / Configuration / Custom Components',
+  component: RenderForm,
+  decorators: [],
+  parameters: {},
+};
+export default meta;
+
+DEFAULT_VALIDATORS[3][1] = 'Vul het verplichte veld in';
+
+const BootstrapInput: React.FC<IComponentProps> = ({callbacks, component, errors, value}) => (
```

Comment:
Web components are supported by React itself. The test case if whether we can inject a custom component into the engine. Not whether React works. Furthermore: The BD has React binding nor (AFAIK) we might/might not use so I'd like to keep this test simple and lightweight.

Reply:
Okay, I was under the impression from the prototypes that there was some additional attention required to bind the event handlers + ensure that changes in values properly cascade down into the low-level webcomponents, which is why I expected these stories.

If you feel sufficiently confident this Just Works :tm: , then I accept that - you know the BD context/requirements better than I do.

Reply:
Additional thought - it might still be useful for documentation purposes to make it clear this is a first class supported API/intended usage? But that could also just be some textual documentation in markdown.

Reply:
It might certainly be the case that certain event handler need to be remapped to the ones that open forms will expose. But whether this happens in a web component or a regular component is merely an abstraction within the implementations domain. 

I agree that we at some point should improve the documentation for actual implementations but I feel it's bit early for that.